Source/Scene/Model.js

```diff
@@ -1454,6 +1454,191 @@ define([
+                    // count property.
+                    //
+                    // https://github.com/KhronosGroup/glTF/issues/258
+
+                    var materialId = primitive.material;
+                    var material = materials[materialId];
+                    var techniqueId = material.technique;
+                    var technique = techniques[techniqueId];
+                    
+                    var programId = technique.program;
+                    if (programId === programName) {
+                        var primitiveAttributes = primitive.attributes;
+                        var attributeVarName;
+                        for (var attributeSemantic in primitiveAttributes) {
+                            if (primitiveAttributes.hasOwnProperty(attributeSemantic)) {
+                                attributeVarName = undefined;
+                                for (var attributeName in technique.parameters) {
+                                    if (technique.parameters.hasOwnProperty(attributeName)) {
+                                        if (defined(attributeVarName)) {
+                                            break;
+                                        }
+                                        if (technique.parameters[attributeName].semantic === attributeSemantic) {
+                                            for (var varName in technique.attributes) {
+                                                if (technique.attributes.hasOwnProperty(varName)) {
+                                                    if (technique.attributes[varName] === attributeName) {
+                                                        attributeVarName = varName;
+                                                        break;
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                                // Skip if the attribute is not used by the material, e.g., because the asset was exported
+                                // with an attribute that wasn't used and the asset wasn't optimized.
+                                if (defined(attributeVarName)) {
+                                    var attributeId = primitiveAttributes[attributeSemantic];
+                                    var a = accessors[attributeId];
+                                    if (a.hasOwnProperty('extensions') && processedAccessors.indexOf(attributeId) < 0) {
+                                        var extensions = a.extensions;
+                                        if (extensions.hasOwnProperty('WEB3D_quantized_attributes')) {
+                                            // Only one of each semantic type can be decoded per shader
+                                            if (decodedAttributes.indexOf(attributeSemantic) >= 0) {
```

Comment:
Add a comment here - or whereever is most appropriate - that there are two approaches: if the quantized attribute has a semantic, the decode matrix is concatenated on the CPU; otherwise, it is done on the GPU by modifying the shader.